packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -18,35 +22,46 @@ class PropertyEditorSidebar extends StatelessWidget {
       children: [
         Text('Property Editor', style: Theme.of(context).textTheme.titleMedium),
         const PaddedDivider.noPadding(),
-        const _PropertiesList(),
+        _PropertiesList(controller: controller),
       ],
     );
   }
 }
 
 class _PropertiesList extends StatelessWidget {
-  const _PropertiesList();
+  const _PropertiesList({required this.controller});
 
-  static const itemPadding = densePadding;
+  final PropertyEditorController controller;
 
+  static const itemPadding = densePadding;
   @override
   Widget build(BuildContext context) {
     // TODO(https://github.com/flutter/devtools/issues/8546) Switch to scrollable
     // ListView when this has been moved into its own panel.
-    return Column(
-      children: <Widget>[
-        ..._properties.map(
-          (property) => _EditablePropertyItem(property: property),
-        ),
-      ].joinWith(const PaddedDivider.noPadding()),
+    return ValueListenableBuilder<List<EditableArgument>>(
+      valueListenable: controller.editableArgs,
+      builder: (context, args, _) {
+        return args.isEmpty
+            ? const Center(
+              child: Text('No widget properties at current cursor location.'),
+            )
+            : Column(
+              children: [
+                for (final property in args)
+                  ...<Widget>[
+                    _EditablePropertyItem(property: property),
+                  ].joinWith(const PaddedDivider.noPadding()),
+              ],
```

Comment:
I doin't think this is doing what we want. Calling joinWith on a list of one item won't add anything. Can we revert to using `map` like we were before?
```
 <Widget>[
        ...args.map(
          (arg) => _EditablePropertyItem(property: arg),
        ),
      ].joinWith(const PaddedDivider.noPadding()),
```

Reply:
oof good catch, switched to calling map